pkgs/development/python-modules/virtualenv/default.nix

```diff
@@ -1,25 +1,53 @@
 { buildPythonPackage
 , fetchPypi
 , lib
-, recursivePthLoader
+, pythonOlder
+, appdirs
+, contextlib2
+, distlib
+, filelock
+, importlib-metadata
+, importlib-resources
+, pathlib2
+, setuptools_scm
+, six
 }:
 
 buildPythonPackage rec {
   pname = "virtualenv";
-  version = "16.7.9";
+  version = "20.0.21";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0d62c70883c0342d59c11d0ddac0d954d0431321a41ab20851facf2b222598f3";
+    sha256 = "1kxnxxwa25ghlkpyrxa8pi49v87b7ps2gyla7d1h6kbz9sfn45m1";
+
   };
 
-  # Doubt this is needed - FRidh 2017-07-07
-  pythonPath = [ recursivePthLoader ];
+  buildInputs = [
+    setuptools_scm
+  ];
 
-  patches = [ ./virtualenv-change-prefix.patch ];
+  propagatedBuildInputs = [
+    appdirs
+    distlib
+    filelock
+    six
+  ] ++ lib.optionals (pythonOlder "3.3") [
+    contextlib2
+  ] ++ lib.optionals (pythonOlder "3.4" && !lib.stdenv.isWindows) [
```

Comment:
There’s no `lib.stdenv`; I think you mean `!stdenv.targetPlatform.isWindows`?

Reply:
ah, so now i took !stdenv.isCygwin is that comparable? I see the hostPlatform is what we are building for? So i should rather take this one.

Reply:
changed to your suggestion in https://github.com/NixOS/nixpkgs/pull/88613/commits/bf3ca289f066e946d5d6ca1cced8bda91f5c0323